SeaPublicWebsite.BusinessLogic/Models/Enums/LoftSpace.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace SeaPublicWebsite.BusinessLogic.Models.Enums
 {
-    public enum AccessibleLoftSpace
+    public enum LoftSpace
```

Comment:
The ticket implies that the answers here should be  `Yes, I have a loft` or `No, I don’t have a loft or my loft has been converted into a room` (and `I'm not sure`)

Reply:
Updated in a separate ticket. Upon merging, the text will be updated.